SPECS/reaper/reaper.spec

```diff
@@ -174,9 +180,16 @@ fi
 %{_unitdir}/cassandra-%{name}.service
 
 %changelog
-* Thu Sep 26 2024 Rohit Rawat <rohitrawat@microsoft.com> - 3.1.1-13
+* Fri Oct 18 2024 Rohit Rawat <rohitrawat@microsoft.com> - 3.1.1-14
 - Patch CVE-2024-45590 in body-parser module
 
+* Thu Oct 15 2024 Rohit Rawat <rohitrawat@microsoft.com> - 3.1.1-13
```

Comment:
This looks wrong, the fast-track change was for release 12, not 13. I think you'd need to move over the changelog below this one as well.

Reply:
It looks like the origin of this issue is that the [original fast-track PR re-ordered the changelog entries](https://github.com/microsoft/azurelinux/pull/10720/files#diff-9b95ff1057d07950c8170737a998746560df54b46beebc4a48c4cc378de9e8f1). We should ideally fix fast-track and then come back to this cherry-pick.